src/ConfirmAttendance/ConfirmAttendance.tsx

```diff
@@ -0,0 +1,118 @@
+import { Box, Flex } from '@rebass/grid';
+import { ErrorMessage, FastField, Formik, FormikProps } from 'formik';
+import * as React from 'react';
+import { boolean, object } from 'yup';
+
+import { Redirect } from 'react-router';
+import { Hacker } from '../api';
+import { FrontendRoute, HackerStatus } from '../config';
+import { Button, H1, H2, MaxWidthBox, Paragraph } from '../shared/Elements';
+import { Form } from '../shared/Form';
+import * as FormikElements from '../shared/Form/FormikElements';
+import ValidationErrorGenerator from '../shared/Form/validationErrorGenerator';
+import WithToasterContainer from '../shared/HOC/withToaster';
+import theme from '../shared/Styles/theme';
+
+interface IConfirmAttendanceProps {
+  status: HackerStatus;
+}
+
+interface IConfirmAttendanceState {
+  confirmed: boolean;
+}
+
+class ManageAccountContainer extends React.Component<
+  IConfirmAttendanceProps,
+  IConfirmAttendanceState
+> {
+  constructor(props: IConfirmAttendanceProps) {
+    super(props);
+    this.state = {
+      confirmed: false,
+    };
+    this.onSubmit = this.onSubmit.bind(this);
+  }
+  public render() {
+    if (this.state.confirmed) {
+      return <Redirect to={FrontendRoute.HOME_PAGE} />;
+    }
+    return (
+      <Flex
+        justifyContent={'center'}
+        alignItems={'center'}
+        flexDirection={'column'}
+      >
+        <MaxWidthBox width={1}>
+          <H1 textAlign={'center'}>Confirm your Attendance</H1>
+          <Paragraph fontSize={'16px'}>
+            We are excited to offer you a spot at McHacks 6 taking place on
```

Comment:
`We are excited to offer you a spot at McHacks 6, taking place on February 2-3, 2019 at Theatre St. James in the Old Port of Montreal.`